kits/automation/time-series-preprocessor/app/globals.css

```
@@ -0,0 +1,15 @@
+@tailwind base;
```

Stylelint (17.7.0)

[error] Unknown at-rule "@tailwind" (at-rule-no-unknown)

(at-rule-no-unknown)